Equinox/Plane.cpp

```diff
@@ -18,12 +18,13 @@ ::Plane::~Plane() {}
 
 void ::Plane::Draw()
 {
+	glDepthMask(GL_FALSE);
```

Comment:
I dont know really if this is necessary. Maybe it's OK to enable it just in case, I'm going to update it.